### PR TITLE
*: cut v0.56.0

### PR DESCRIPTION
**Pull request by simonpasquier**
## Description

Cutting v0.56.0...

## Type of change

_What type of changes does your code introduce to the Prometheus operator? Put an `x` in the box that apply._

- [ ] `CHANGE` (fix or feature that would cause existing functionality to not work as expected)
- [ ] `FEATURE` (non-breaking change which adds functionality)
- [ ] `BUGFIX` (non-breaking change which fixes an issue)
- [ ] `ENHANCEMENT` (non-breaking change which improves existing functionality)
- [X] `NONE` (if none of the other choices apply. Example, tooling, build system, CI, docs, etc.)

## Changelog entry

_Please put a one-line changelog entry below. This will be copied to the changelog file during the release process._

```release-note

```

### CODE REVIEW

**Comment by simonpasquier** [comment]
Waiting for the CI on https://github.com/prometheus-operator/prometheus-operator/pull/4738 to see if we can include it in this release.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,14 @@
+## 0.56.0 / 2022-04-20
+
+* [CHANGE] Added validation at the API level for duration and time-based fields of the Prometheus CRD. #4684
```
Comment (slashpai) [inline CHANGELOG.md:3]: Missed validation for size based retention and log settings :) 
https://github.com/prometheus-operator/prometheus-operator/pull/4661
https://github.com/prometheus-operator/prometheus-operator/pull/4638

Comment (simonpasquier) [reply CHANGELOG.md:5]: thanks! not sure how I missed them :)

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,16 @@
...
+* [CHANGE] Added shortnames for custom resources (`amcfg` for AlertmanagerConfig, `am` for Alertmanager, `pmon` for PodMonitor, `prb` for Probe, `prom` for Prometheus, `smon` for ServiceMonitor, `ruler` for Thanos Ruler). #4680
+* [FEATURE] Added `status` subresource to the Prometheus CRD. #4580
+* [ENHANCEMENT] Added `excludedFromEnforce` field to the Prometheus CRD. It allows to define PodMonitor, ServiceMonitor, Probe or PrometheusRule objects for which the namespace label enforcement (if enabled) should not be applied. This deprecates `prometheusRulesExcludedFromEnforce` which is still supported but users are encouraged to migrate to the new field. #4397
+* [ENHANCEMENT] Added `enableRemoteWriteReceiver` field to the Prometheus CRD. #4633
+* [ENHANCEMENT] Added `entity` and `actions` fields for the OpsGenie receiver to the AlertmanagerConfig CRD. #4697
+* [ENHANCEMENT] Added `prometheus_operator_reconcile_duration_seconds` histogram metric. #4706
+* [BUGFIX] Added support for `opsgenie_api_key_file` and `api_key_file` in the generated Alertmanager configuration. #4666 #4738
```
Comment (fpetkovski) [inline CHANGELOG.md:12]: Is this a feature maybe?

Comment (simonpasquier) [reply CHANGELOG.md:12]: I hesitated because #4714 (fixed by #4738) was filed as a bug but it could be [ENHANCEMENT/BUGFIX] indeed :)

Comment (simonpasquier) [reply CHANGELOG.md:12]: @fpetkovski given that #4714 was reported as a bug, I'd be inclined to keep it as a [BUG]. WDYT?

Comment (fpetkovski) [reply CHANGELOG.md:12]: Sounds good to me :)

**Review by fpetkovski — APPROVED** [review]

**Pull request merged**